finagle-core/src/main/scala/com/twitter/finagle/ssl/server/SslServerSessionVerifier.scala

```diff
@@ -45,7 +46,8 @@ object SslServerSessionVerifier {
    * verifies every given session.
    */
   val AlwaysValid: SslServerSessionVerifier = new SslServerSessionVerifier {
-    def apply(address: Address, config: SslServerConfiguration, session: SSLSession): Boolean = true
+    def apply(address: Address, config: SslServerConfiguration, session: SSLSession): Future[Boolean] =
```

Comment:
cols < 100

Reply:
what does the style guide suggests about method declaration? how should I split it to multiple lines?

Is the following okay?
```scala
def apply(
  address: Address,
  config: SslServerConfiguration,
  session: SSLSession): Future[Boolean] =
{
  ???
}
```

Reply:
Thanks for asking. For us it would look like ...

    def apply(
        address: Address,
        config: SslServerConfiguration,
        session: SSLSession
    ): Future[Boolean] = {
       // More Here
    }